settings.gradle

```diff
@@ -14,18 +14,18 @@ dependencyResolutionManagement {
 }
 
 rootProject.name = 'dai-deployment-templates'
-include 'aws-lambda-scorer:lambda-template'
-include 'aws-lambda-scorer:terraform-recipe'
-include 'common:jdbc'
+//include 'aws-lambda-scorer:lambda-template'
```

Comment:
All deprecated scorer which not only takes long time to build also no longer used by MLOps